src/components/global/Playground/index.tsx

```diff
@@ -76,7 +85,11 @@ export default function Playground({
     <div className="playground">
       <div className="playground__container">
         <div className="playground__control-toolbar">
-          {/* TODO FW-742: Code language switcher */}
+          <div className="playground__control-group">
+            {Object.values(UsageTarget).map(lang => (
+              <CodeBlockButton language={lang} usageTarget={usageTarget} setUsageTarget={setUsageTarget} />
```

Comment:
If I click "Vue", this sets the active framework to "Vue", but the code snippet key is passed in as "vue" so the code is never actually shown.

Can we change this so that the key is lower case?

Reply:
The `CodeBlockButton` component uses `UsageTarget[language]` (the lowercase version) for setting `usageTarget`. The key (the uppercase one) is only used for the display. (I pushed a couple commits right as you were reviewing, so that change might've gotten missed)